pandas/io/tests/test_packers.py

```diff
@@ -543,25 +543,6 @@ def test_sparse_frame(self):
         self._check_roundtrip(ss3, tm.assert_frame_equal,
                               check_frame_type=True)
 
-    def test_sparse_panel(self):
-
```

Comment:
As I mentioned <a href="https://github.com/pydata/pandas/pull/13778#discussion_r71992701">below</a>, I don't believe that there are any pickles for `SparsePanel`.